test/test_boost_hash.cpp

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (C) 2019 James E. King III
+//
+// Permission to copy, use, modify, sell and distribute this software
+// is granted provided this copyright notice appears in all copies.
+// This software is provided "as is" without express or implied
+// warranty, and with no claim as to its suitability for any purpose.
+//
+// Distributed under the Boost Software License, Version 1.0. (See
+// accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#include <boost/config.hpp>
+#include <boost/dynamic_bitset.hpp>
+#include <boost/detail/lightweight_test.hpp>
```

Comment:
This is the deprecated header. Should include <boost/core/lightweight_test.hpp>

Reply:
Thanks, fixed in the next push.